app/services/mailer/views/confirmation.handlebars

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Launch Online Application</title>
+
+  <style>
```

Comment:
can we avoid using embedded styles in email HTML templates?
https://content.myemma.com/blog/css-in-html-emails-what-you-need-to-know-to-get-started